cloudinit/sources/DataSourceOpenNebula.py

```diff
@@ -350,7 +350,8 @@ def varprinter(vlist):
     # exclude vars in bash that change on their own or that we used
     excluded = (
         "EPOCHREALTIME", "EPOCHSECONDS", "RANDOM", "LINENO", "SECONDS", "_",
-        "__v")
+        "__v", "SRANDOM",
+    )
```

Comment:
I'd flip the order of these two things.
Only because __v is "our" variable, where SRANDOM i'm guessing is a shell one.

other than that... i'd think it should be OK.

Reply:
Good catch, fixed!